src/java.desktop/windows/native/libawt/windows/ThemeReader.cpp

```diff
@@ -236,18 +238,20 @@ static void assert_result(HRESULT hres, JNIEnv *env) {
  * Signature: (Ljava/lang/String;)J
  */
 JNIEXPORT jlong JNICALL Java_sun_awt_windows_ThemeReader_openTheme
-(JNIEnv *env, jclass klass, jstring widget) {
+(JNIEnv* env, jclass klass, jstring widget, jint dpi) {
 
-    LPCTSTR str = (LPCTSTR) JNU_GetStringPlatformChars(env, widget, NULL);
+    LPCTSTR str = (LPCTSTR)JNU_GetStringPlatformChars(env, widget, NULL);
```

Comment:
I'd rather keep the original formatting for casts as well as for `JNIEnv *env` in the declaration.